src/warnet/control.py

```diff
@@ -123,7 +123,7 @@ def down():
     tanks = get_mission("tank")
     with console.status("[yellow]Uninstalling tanks...[/yellow]"):
         for tank in tanks:
-            cmd = f"helm uninstall {tank.metadata.name}"
+            cmd = f"helm uninstall {tank.metadata.name} --namespace {get_default_namespace()}"
```

Comment:
I think this is fine for now, but leaving a comment for future self that we will need to rethink this if we are deploying commanders into a separate namespace.

Reply:
@josibake I'm going to merge then in order to get `warcli down` working.

As for commanders, perhaps we could concat get_default_namespace with '-commander', and place them in that namespace.